code/modules/mob/mob_defines.dm

```diff
@@ -180,4 +180,6 @@
 	var/list/progressbars = null
 
 
-	var/speed_factor = 1.0
\ No newline at end of file
+	var/speed_factor = 1.0
+
+	var/datum/stat_holder/stats
```

Comment:
Wasnt this in living defines too?

Reply:
not anymore

Reply:
But why?

Reply:
its done in order to allow stat checks for usr types which are mobs